src/engine/renderer/glsl_source/fogQuake3_fp.glsl

```diff
@@ -24,7 +24,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 #define FOGQUAKE3_GLSL
 
-uniform sampler2D	u_ColorMap;
+uniform sampler2D	u_FogMap;
```

Comment:
What do you mean?

Reply:
The `uniform sampler2D u_FogMap;`, actually that's a tab and not many spaces, but fixing the ugliness of the code when modifying is the best way to prevent noisy restyling commits. Nothing is blocking there, just noticing the code can be beautified at the same time and that would save us a noisy restyle commit.

Reply:
Alright, done.